src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -579,5 +579,37 @@ public static string GetNativeLibsRootDirectoryPath (string androidBinUtilsDirec
 			string relPath = GetToolsRootDirectoryRelativePath (androidBinUtilsDirectory);
 			return Path.GetFullPath (Path.Combine (androidBinUtilsDirectory, relPath, "lib"));
 		}
+
+		public static string? GetAssemblyCulture (ITaskItem assembly)
+		{
+			// The best option
+			string? culture = assembly.GetMetadata ("Culture");
+			if (!String.IsNullOrEmpty (culture)) {
+				return TrimSlashes (culture);
```

Comment:
nit: it'd be pretty weird if Culture contained any slashes, might be ok to return it unmodified

Reply:
I learned the hard way... :)